addons/mass_mailing/static/src/js/mass_mailing_form.js

```diff
@@ -1,10 +1,107 @@
 /** @odoo-module **/
 
 import FormView from 'web.FormView';
+import FormController from 'web.FormController';
 import FormRenderer from 'web.FormRenderer';
+import { bus, _t } from 'web.core';
 import viewRegistry from 'web.view_registry';
 import config from 'web.config';
 
+const MassMailingFormController = FormController.extend({
+    /**
+     * @override
+     */
+    init() {
+        this._super(...arguments);
+        this._boundTriggerIframeUpdated = this._triggerIframeUpdated.bind(this);
+        this._boundInitIframeResizer = this._initIframeResizer.bind(this);
+        bus.on('DOM_updated', this, this._boundTriggerIframeUpdated);
+        bus.on('iframe_updated', this, this._boundInitIframeResizer);
+    },
+    /**
+     * @override
+     */
+    destroy() {
+        bus.off('DOM_updated', this, this._boundTriggerIframeUpdated);
+        bus.off('iframe_updated', this, this._boundInitIframeResizer);
+        this._super(...arguments);
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * Resize the given iframe so its height fits its contents and initialize a
+     * resize observer to resize on each size change in its contents.
+     *
+     * @param {JQuery} $iframe
+     */
+    _initIframeResizer($iframe) {
+        if (!$iframe.length || !$iframe.contents().length) {
+            return;
+        }
+        this._autoresizeIframe($iframe);
+        const $iframeDoc = $iframe.contents();
+        const $ref = $iframeDoc.find('#iframe_target');
+        if ($ref[0]) {
+            const isFull = window.top.document.body.classList.contains("o_field_widgetTextHtml_fullscreen");
+            $ref.css({
+                display: isFull ? '' : 'flex',
+                'flex-direction': isFull ? '' : 'column',
+            });
+            if (this._resizeObserver) {
+                this._resizeObserver.disconnect();
+            }
+            this._resizeObserver = new ResizeObserver(() => this._autoresizeIframe($iframe));
+            this._resizeObserver.observe($ref[0]);
+        }
+        const $content = this.$('.o_content');
+        $content.on('scroll', () => this._repositionSidebar($iframe));
```

Comment:
Does this mean that you register a scroll listener every time the "_initIframeResizer" is called? 
Doesn't that end up creating a lot of unnecessary / redundant ones?

Another approach would be to register a scroll listener at the controller level:

```js
events: _.extend({}, FormController.prototype.events, {
    'scroll .o_content': '_onContentScroll',
}
```

And in the "_initIframeResizer", you would update a "this.$iframe = $iframe" reference, to be able to access it in the listener.
(Just some suggestion, maybe it wouldn't work, I don't know the code enough).